task/07-yield-tasks.js

```diff
@@ -46,9 +58,16 @@ function* get99BottlesOfBeer() {
  *
  */
 function* getFibonacciSequence() {
-  throw new Error('Not implemented');
-}
+  let n = 39;
+  const infinite = !n && n !== 0;
+  let current = 0;
+  let next = 1;
 
+  while (infinite || n--) {
+    yield current;
+    [current, next] = [next, current + next];
+  }
+}
 
```

Comment:
you should learn this solution 
`yield 0;
  yield 1;
  let a = 0;
  let b = 1;
  while(true){
    const c = a + b;
    a = b;
    b = c;
    yield c;
  }`